mlflow/server/handlers.py

```diff
@@ -27,9 +29,13 @@ def _get_store():
     from mlflow.server import FILE_STORE_ENV_VAR, ARTIFACT_ROOT_ENV_VAR
     global _store
     if _store is None:
-        store_dir = os.environ.get(FILE_STORE_ENV_VAR, os.path.abspath("mlruns"))
-        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, store_dir)
-        _store = FileStore(store_dir, artifact_root)
+        store_uri = os.environ.get(FILE_STORE_ENV_VAR, os.path.abspath("mlruns"))
+        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, store_uri)
```

Comment:
is this line only required if its a FileStore? ... redundant if dynamodb?

Reply:
I've tidied this up to use the `_is_dynamodb_uri`  and `_get_dynamodb_store` methods from tracking client.